### ISSUE TITLE
Problematic protein identification with viewer.expandAndCollapseSelection(proteinIdentifier)

### ISSUE THREAD

**Issue by EliotRagueneau** [MICommunity/ComplexViewer#160]
Currently, `viewer.expandAndCollapseSelection(proteinIdentifier)` uses the UniProt id to collapse or expand the proteins, but this is problematic on cases such as [this one on intact-portal](https://intact-portal.github.io/intact-portal-view/details/interaction/EBI-7156290) because both participants have the same identifier, but they are not the same participants.

Would there be a way to use IntAct participant's ac for this function instead, to avoid this problem? Or maybe se should add another parameter indicating which field complex viewer should be looking for among the participants to collapse/expand them? e.g. `viewer.expandAndCollapseSelection(id, idType)`

P.S.: I'm relatevily new to IntAct team, and began recently to work a bit on complex viewer, but you've done a really good job here @colin-combe, it's a really nice viewer. Thanks a lot for your help and support!

**Comment by colin-combe** [comment]
`viewer.expandAndCollapseSelection(id, idType)`

yes, we can do that

**Comment by colin-combe** [comment]
@EliotRagueneau - hmm, I don't think I can find this _Intact participant ac_ in the json the viewer recieves. 
To see the json, I'm looking here: https://wwwdev.ebi.ac.uk/intact/ws/graph/interaction/export/EBI-7156290
It may need a change to the json? (or do you see it in there somewhwhere?)

**Comment by EliotRagueneau** [comment]
You are absolutely right, we'll have to fix this! @noedelta maybe you could do that since you're currently working on the Complex Viewer?

**Comment by colin-combe** [comment]
i'll unassign myself since it can't be done until _Intact participant ac_ is in the json

**Comment by colin-combe** [comment]
from meeting, plan is for @noedelta to add  _Intact participant ac_ to the json
